include/EASTL/type_traits.h

```diff
@@ -865,7 +865,10 @@ namespace eastl
 	// 
 	///////////////////////////////////////////////////////////////////////
 
-	#if defined(EA_COMPILER_NO_VARIADIC_TEMPLATES) || defined(EA_COMPILER_MSVC) // VS2013 fails to compile the variadic code below due to what looks like a deficiency in their handling of integral variadic template parameters.
+	// VS2013 fails to compile the variadic code below due to what looks like a deficiency in their handling of integral variadic template parameters.
+	// Also mingw's clang and gcc fail to compile the code (on windows, linux untested).
+	#if defined(EA_COMPILER_NO_VARIADIC_TEMPLATES) || defined(EA_COMPILER_MSVC) || \
```

Comment:
Since I flagged other edits in this PR...If you could place this all on a single line that would be preferable.

Thanks.